omero/sysadmins/index.rst

```diff
@@ -54,11 +54,12 @@ an existing component, or just looking for more background information, there
 is a section about the server within the :doc:`/developers/index`;
 the best starting point is the :doc:`/developers/Server` for developers.
 
+******************
 Background reading
```

Comment:
```suggestion
Requirements, examples, limitations
```

Reply:
Two other options would be to split into separate sections or move these pages at the top-level of the hierarchy (i.e. don't try and nest them under a heading as we can't rely define it)